src/site/stages/build/drupal/graphql/GetAllPages.graphql.js

```diff
@@ -97,7 +97,7 @@ const buildQuery = ({ useTomeSync }) => {
   const nodeQuery = useTomeSync
     ? ''
     : `
-    nodeQuery(limit: 5000, filter: {
+    nodeQuery(limit: 15000, filter: {
```

Comment:
Probably OK to merge to master, but we may run into memory issues.

Reply:
I'll hold this one back